openedx-commerce.php

```diff
@@ -1,11 +1,11 @@
 <?php
 /**
  * Plugin Name:       Open edX Commerce
- * Plugin URI:        https://github.com/eduNEXT/openedx-wordpress-ecommerce
+ * Plugin URI:        https://github.com/openedx/openedx-wordpress-ecommerce
  * Description:       Easily connect your WooCommerce store to Open edX.
  * Version:           2.0.3
- * Author:            eduNEXT
- * Author URI:        https://edunext.co/
+ * Author:            The Open edX Community
```

Comment:
This two I would leave as edunext. I think "the open edx community" is two broad and lacks accountability.